addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -1902,8 +1900,8 @@ public Action:StartResponseTimer(Handle:timer)
 	decl String:sound[PLATFORM_MAX_PATH];
 	if(FindSound("begin", sound, sizeof(sound)))
 	{
-		EmitSoundToAllExcept(FF2SOUND_MUTEVOICE, sound, Boss[0]);
```

Comment:
Why were these removed?

Reply:
This one I specifically left because it's the intro sound, with Boss[0] argument the sound cannot be heard. Without it, it can be heard by all players. I already explained this in steam messages i sent you